### PR TITLE
Adding the FR version of Advanced Mapping in JOSM

### PR DESCRIPTION
**Pull request by Sophie-GelinasGagne**

### CODE REVIEW

**Comment by danbjoseph** [comment]
similar to https://github.com/MissingMaps/missingmaps.github.io/commit/d88904ed6d008c6bbb10ddf3bc859576d7c2e3d0  this will require some additional changes to:
-  https://github.com/MissingMaps/missingmaps.github.io/blob/publish/gulpfile.js
- https://github.com/MissingMaps/missingmaps.github.io/blob/publish/app/_includes/advanced.html#L14
- and the translation files in https://github.com/MissingMaps/missingmaps.github.io/tree/publish/app/_data



i can help if needed, but not for probably several days.

**Comment by Sophie-GelinasGagne** [comment]
Noted! I'll look into it and let you know if I have any questions. Thanks

**Comment by Sophie-GelinasGagne** [comment]
@danbjoseph As per my [comment](https://github.com/MissingMaps/missingmaps.github.io/commit/d88904ed6d008c6bbb10ddf3bc859576d7c2e3d0#commitcomment-81949055), I changed the names of some of the functions in the [gulpfile.js](https://github.com/MissingMaps/missingmaps.github.io/blob/publish/gulpfile.js) Let me know if that does not work.

**Comment by danbjoseph** [comment]
i think this looks good. and yes, you're right that the function names needed an update based on evolving content of the files.

File: app/_includes/advanced.html [hunk 1]
```diff
@@ -11,7 +11,7 @@ <h1 class="title feature-header">{{site.data[locale].advanced.title}}</h1>
   
   <div class="row contribute-options">
     <div class="columns" style="text-align: center">
-      <a href="{{ site.baseurl }}/assets/downloads/JOSM_Advanced_Mapping_EN.pdf">
+      <a href="{{ site.baseurl }}/assets/downloads/{{site.data[locale].checker.links.advancedJOSM_pdf:}}">
```
Comment (danbjoseph) [inline app/_includes/advanced.html:14]: i didn't catch this before merging but the `:` at the end inside the `}}` might cause a problem

File: app/_data/fr.yml [hunk 2]
```diff
@@ -189,7 +189,9 @@ beginner:
 ##############
 advanced:
   title: "CARTOGRAPHIE AVANCÉE"
-  guide: "JOSM Guide"
+  links:
+    guide: "Manuel de cartographie avancée avec JOSM"
+    advancedJOSM_pdf: "JOSM_Advanced_Mapping_EN.pdf"
```
Comment (danbjoseph) [inline app/_data/fr.yml:194]: and i think this needs `_FR` swapped into the filename

**Pull request merged**